soap_test.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# Try to dump SOAP vectors (features) for each atom of each molecule
+
+import sys
+import numpy as np
+from ase.io import read as ase_read
+from rascal.representations import SphericalInvariants as SOAP
+
+# What is this integer? Unit? Reasonable range to explore?
```

Comment:
that's the number of radial basis, i.e. the radial resolution. Typical range is 4-9, GTO can go up to 22, DVR is mostly unlimited.